packages/web-core/src/services/ConnectService.ts

```diff
@@ -179,28 +180,48 @@ export class ConnectService {
   }
 
   async login(identifier: string, source: PasskeyLoginSource): Promise<Result<ConnectLoginFinishRsp, CorbadoError>> {
+    const resStart = await this.loginStart(identifier, source);
+
+    return this.continueLogin(resStart);
+  }
+
+  async loginStart(
+    identifier: string,
+    source: PasskeyLoginSource,
+  ): Promise<Result<ConnectLoginStartRsp, CorbadoError>> {
     const existingProcess = ConnectProcess.loadFromStorage(this.#projectId);
     if (!existingProcess) {
       return Err(CorbadoError.missingInit());
     }
 
-    const resStart = await this.wrapWithErr(() =>
+    const res = await this.wrapWithErr(() =>
       this.#connectApi.connectLoginStart({
         identifier,
         source: source as ConnectLoginStartReqSourceEnum,
       }),
     );
-    if (resStart.err) {
+    if (res.err) {
       ConnectLastLogin.clearStorage(this.#projectId);
-      return resStart;
+      return res;
     }
 
-    if (!resStart.val.assertionOptions) {
+    if (!res.val.assertionOptions) {
       ConnectLastLogin.clearStorage(this.#projectId);
       return Err(CorbadoError.noPasskeyAvailable());
     }
 
+    return res;
+  }
+
+  async continueLogin(
```

Comment:
if the first method is called "loginStart" I would call the second method "loginFinish" for more consistency